frontend/src/components/pages/CamperRefundCancellation/index.tsx

```diff
@@ -61,13 +95,36 @@ const CamperRefundCancellation = (): React.ReactElement => {
         />
         <Box pr={{ lg: "30%", sm: "12%", md: "12%" }} pl="12%" pt="5%">
           <Box pb="40px">
-            <Text as="b" mt="70px" fontSize="40px">
-              {campName}
+            <Text
+              as="b"
```

Comment:
Is this still necessary with `textStyle`? (other instances of same thing in this file too)

Reply:
Good catch, I removed it